new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -363,4 +823,267 @@ private void verifySampleAppAccessThroughNginx() {
+   * @return the operator Helm installation parameters
+   */
+  private static HelmParams installOperatorHelmChart(String operNamespace,
+                                                    String opServiceAccount,
+                                                    boolean createOpSA,
+                                                    boolean withRestAPI,
+                                                    boolean createSecret,
+                                                    String errMsg,
+                                                    String helmStatus,
+                                                    int externalRestHttpsPort,
+                                                    HelmParams opHelmParams,
+                                                    boolean elkIntegrationEnabled,
+                                                    String... domainNamespace) {
+    LoggingFacade logger = getLogger();
+    String opReleaseName = opHelmParams.getReleaseName();
+
+    if (createOpSA) {
+      // Create a service account for the unique operNamespace
+      logger.info("Creating service account");
+      assertDoesNotThrow(() -> createServiceAccount(new V1ServiceAccount()
+          .metadata(new V1ObjectMeta()
+              .namespace(operNamespace)
+              .name(opServiceAccount))));
+      logger.info("Created service account: {0}", opServiceAccount);
+    }
+
+    // get operator image name
+    String operatorImage = getOperatorImageName();
+    assertFalse(operatorImage.isEmpty(), "operator image name can not be empty");
+    logger.info("operator image name {0}", operatorImage);
+    if (createSecret) {
+      boolean secretExists = false;
+      V1SecretList listSecrets = listSecrets(operNamespace);
+      if (null != listSecrets) {
+        for (V1Secret item : listSecrets.getItems()) {
+          if (item.getMetadata().getName().equals(REPO_SECRET_NAME)) {
+            secretExists = true;
+            break;
+          }
+        }
+      }
+      if (!secretExists) {
+
+        // Create Docker registry secret in the operator namespace to pull the image from repository
+        logger.info("Creating Docker registry secret in namespace {0}", operNamespace);
+        createDockerRegistrySecret(operNamespace);
+      }
+    }
+    // map with secret
+    Map<String, Object> secretNameMap = new HashMap<>();
+    secretNameMap.put("name", REPO_SECRET_NAME);
+
+    // operator chart values to override
+    OperatorParams opParams = new OperatorParams()
+        .helmParams(opHelmParams)
+        .imagePullSecrets(secretNameMap)
+        .domainNamespaces(java.util.Arrays.asList(domainNamespace))
+        .serviceAccount(opServiceAccount);
+
+    // use default image in chart when repoUrl is set, otherwise use latest/current branch operator image
+    if (opHelmParams.getRepoUrl() == null) {
+      opParams.image(operatorImage);
+    }
+
+    // enable ELK Stack
```

Comment:
Since we are not using Elk stack here, make the local utility method simple when possible.

Reply:
done